src/AppInstallerCommonCore/Manifest/ManifestYamlPopulator.cpp

```diff
@@ -664,9 +671,15 @@ namespace AppInstaller::Manifest
         return resultErrors;
     }
 
-    ValidationErrors ManifestYamlPopulator::PopulateManifestInternal(const YAML::Node& rootNode, Manifest& manifest, const ManifestVer& manifestVersion, bool fullValidation)
+    ValidationErrors ManifestYamlPopulator::PopulateManifestInternal(
+        const YAML::Node& rootNode,
+        Manifest& manifest,
+        const ManifestVer& manifestVersion,
+        bool fullValidation,
+        bool errorOnVerifiedPublisherFields)

```

Comment:
For the long term it probably makes more sense to pass the `ManifestValidateOption` around rather than individual bools.  That would also mean moving it into the `Manifest` namespace rather than being in YAML (which makes sense really).

Reply:
When I wrote that I thought that it was a struct; I still stand by it being easier to update though.  You only need to add a new helper function (if that even) to call `WI_IsFlagSet` and you have a new control.